### PR TITLE
Add remaining Board API endpoints

### PR DESCRIPTION
**Pull request by trevorbayless**
Adds the remaining Board API endpoints:

- `/api/board/game/{game_id}/chat`
- `/api/board/game/{game_id}/claim-victory`
- `/api/board/game/{game_id}/berserk`

### CODE REVIEW

**Comment by trevorbayless** [comment]
PR reformatted with black

File: berserk/clients.py [hunk 1]
```diff
@@ -1084,6 +1094,30 @@ def decline_takeback(self, game_id):
...
+        time has elapsed.
+
+        :param str game_id: ID of an in-progress game
+        :return: True if successful
+        :rtype: bool
+        """
+        path = f"/api/board/game/{game_id}/claim-victory/"
+        return self._r.post(path)["ok"]
+
+    def berserk_tournament_game(self, game_id):
```
Comment (kraktus) [inline berserk/clients.py:1111]: maybe `go_berserk` instead? No strong feeling, but currently feels a bit long name. `berserk` alone could be ambiguous given the lib name hence the `go_`

Comment (trevorbayless) [reply berserk/clients.py]: Agreed. Updated to `go_berserk`.

**Comment by trevorbayless** [comment]
Added type hints

**Pull request merged**